src/components/result-ui/index.js

```diff
@@ -7,11 +7,15 @@ import Summary from "./summary";
 const ResultUI = () => {
   const { responseDocument } = useContext(LocationContext);
   const data = responseDocument ? responseDocument.getData() : [];
+  const isCollection = responseDocument && responseDocument.isCollectionDocument();
 
   return (
     <>
       <DisplayRaw
-        title="Results"
+        title={isCollection
+          ? data.length ? `Results (${data.length})` : 'No results'
```

Comment:
Do you think the title should be concat differently so that the "tab" title can always show `Results (n)` ?
![image](https://user-images.githubusercontent.com/8967332/60354433-6ffe0b00-9989-11e9-974f-a7243c65b2cc.png)

Reply:
Eww, you're right. I didn't realize that "No results" was getting shoved in the tab like that.

Reply:
I think it's ok for now tho, I'll revisit this with the results theming